src/register/data/tests/reducers.test.js

```diff
@@ -63,6 +64,37 @@ describe('Registration Reducer Tests', () => {
       },
     );
   });
+  it('on regirstration success call', () => {
+    const payload = {
+      redirectUrl: 'http://localhost:18000/dashboard',
```

Comment:
```suggestion
      redirectUrl: `${getConfig().BASE_URL}/dashboard`,
```

Reply:
addressed

Reply:
Sorry, I missed 1 thing. Could you please import dashboard from constants